packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -62,6 +65,12 @@ const optionSets = {
     [ExampleOptionsType.manyOptions]: manyOptions
 } as const;
 
+const filterModeDescription = `
```

Comment:
We should provide usage guidance about when to set this. I think @fredvisser mentioned in a comment on a previous PR/issue that we'd likely recommend "none" if there are few items and "standard" if there are many items but you'd have to dig that up to find the exact numbers.

Reply:
Updated docs.